workbench/_api/metadata.py

```
@@ -0,0 +1,363 @@
+        or hasattr(config, "image_token_index")
+        or hasattr(config, "image_token_id")
+        or getattr(config, "model_type", None) in VISION_MODEL_TYPES
+    ):
+        raise UnsupportedModel(f"{model_name} has a vision component ({config.model_type})")
+
+    n_layers = getattr(config, "num_hidden_layers", None) or getattr(config, "n_layer", 0)
+
+    # Parameter count from safetensors header (no weights download)
+    try:
+        num_params = _get_param_count(model_name)
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)